app/assets/javascripts/tictactoe.js

```diff
@@ -1 +1,128 @@
-// Code your JavaScript / jQuery solution here
+var turn = 0;
+var currentGame = 0;
+
+var  winCombinations = [
+  [0,1,2],
+  [3,4,5],
+  [6,7,8],
+  [0,3,6],
+  [1,4,7],
+  [2,5,8],
+  [0,4,8],
+  [2,4,6]
+]
+
+function player() {
+  return turn %2 === 0 ? 'X':'O';
```

Comment:
A cool JS trick: 0 is a "falsy" value, so this could also read `var player = () => turn % 2 ? 'O' : 'X';`